docs/source/tutorials/getting-started/lifecycling.rst

```diff
@@ -0,0 +1,153 @@
+
+* The :ref:`Distribution Rule <module-daml-finance-lifecycle-rule-distribution-35531>` defines the business logic to calculate the resulting
+  lifecycle effect from a given distribution event. It is signed by the `Bank` as a provider.
+* The :ref:`Claim Rule <module-daml-finance-lifecycle-rule-claim-99318>` allows a holder of the target instrument to claim the effect
+  resulting from the distribution event. By presenting their holding they can instruct the settlement of the holding transfers described in the effect.
+
+We then create a distribution event describing the terms of the dividend to be payed.
+
+.. literalinclude:: ../../../code-samples/getting-started/daml/Scripts/Lifecycling.daml
+  :language: daml
+  :start-after: -- CREATE_EVENT_BEGIN
+  :end-before: -- CREATE_EVENT_END
+
+Now we can process the distribution event using the distribution rule.
+
+.. literalinclude:: ../../../code-samples/getting-started/daml/Scripts/Lifecycling.daml
+  :language: daml
+  :start-after: -- LIFECYCLE_EVENT_BEGIN
+  :end-before: -- LIFECYCLE_EVENT_END
+
+The result of this is an effect describing the per-unit asset movements to be executed for token holders. Each holder can now present their holding
+to *claim* the effect and instruct settlement of the associated entitlements.
+
+.. literalinclude:: ../../../code-samples/getting-started/daml/Scripts/Lifecycling.daml
+  :language: daml
+  :start-after: -- CLAIM_EVENT_BEGIN
+  :end-before: -- CLAIM_EVENT_END
+
+As a side-effect of claiming the effect the presented holding is exchanged for the new token version. This is to prevent a holder from claiming a
+given effect twice.
+
+In our example of a cash dividend only a single instruction is generated: the movement of cash from the bank to
+the token holder. This instruction along with its batch is now settled the usual way, as described in the previous :doc:`Settlement <settlement>` tutorial.
+
+.. literalinclude:: ../../../code-samples/getting-started/daml/Scripts/Lifecycling.daml
+  :language: daml
+  :start-after: -- EFFECT_SETTLEMENT_BEGIN
+  :end-before: -- EFFECT_SETTLEMENT_END
+
+Note that the bank in this case does not actually transfer the cash from another account, but simply credits Bob's account by using the ``CreditReceiver``
+allocation type. In a real-world dividend scenario one would additionally model the flow of funds from the issuer to the bank using the same lifecycle
+process as described above.
+
+Frequently Asked Questions
+**************************
+
+Which party should create and sign the lifecycle rules and events?
+==================================================================
+
+In the simplified scenario for this tutorial we have used the bank as both the *issuer* and *depository* for the instruments involved. In a real-world case
+instruments and their corresponding lifecycle rules and events would be maintained by an actual issuer, with the depository acting as a 3rd-party trust anchor.
+
```

Comment:
Not sure if this is something that we want to describe here, but what happens in the case of an interest rate swap between two counterparties (and no clear issuer)?
Should one counterparty take the role of the issuer?

Reply:
Yes, we should explain this somewhere but maybe not here in the getting-started.